tests/unit/StartDate.spec.ts

```diff
@@ -0,0 +1,85 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { createLocalVue, createWrapper, mount } from '@vue/test-utils'
+import VueRouter from 'vue-router'
+import mockRouter from './MockRouter'
+import { getVuexStore } from '@/store/'
+import { StartDate } from '@/components/common/YourCompany'
+import { DatePicker } from '@bcrs-shared-components/date-picker'
+import flushPromises from 'flush-promises'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const localVue = createLocalVue()
+localVue.use(VueRouter)
+
+let router: any = mockRouter.mock()
+let store: any = getVuexStore()
+
+
+describe('Start Date', () => {
+  let wrapper: any
+
+  let correctionInformation = {
```

Comment:
Use const if you're not reassigning this variable. Ditto on lines 17-18.

(This won't make any difference to how the code runs but it might prevent a developer from reassigning the variables without thinking it through.)